setup.py

```diff
@@ -16,6 +16,7 @@
         "parsel >= 1.5.0",
         "scrapy >= 2.6.0",
         "sqlitedict >= 1.5.0",
+        "twisted >= 18.9.0",
```

Comment:
I realized we are using it directly in scrapy_poet.middlewares, so I think it best to include it here. I used the minimum version from Scrapy, since Scrapy is a dependency as well and using an older version will not be possible, but we only use inlineCallbacks and Deferred, so if it were not for Scrapy we could probably support much older versions.